Formula/shogun.rb

```diff
@@ -0,0 +1,151 @@
+class Shogun < Formula
+  desc "Large scale machine learning toolbox"
+  homepage "http://www.shogun-toolbox.org/"
+  url "http://shogun-toolbox.org/archives/shogun/releases/6.1/sources/shogun-6.1.3.tar.bz2"
+  sha256 "57169dc8c05b216771c567b2ee2988f14488dd13f7d191ebc9d0703bead4c9e6"
+
+  bottle do
+    sha256 "06aafb46d4cd4e53ebed6f8bb32e1279b5a8b5de9aae2359261cee9de87fc2ca" => :sierra
+    sha256 "e5dd5b447ef40488fb32fa36e2eb943975a6b28db8ae27859e39cb8a87a077a9" => :el_capitan
+    sha256 "b3d320d345da09b55ffb98a59f4fac3b2f6c2672fcbcda693fe0e7a9ee4795a3" => :yosemite
+    sha256 "dfbd03cba5e1a134a520d6c06aceaa3c5143cad638fc208150d980a44e5252cf" => :x86_64_linux
+  end
+
+  patch :DATA
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "swig" => :build
+  depends_on "arpack"
+  depends_on "eigen"
+  depends_on "glpk"
+  depends_on "hdf5"
+  depends_on :java
```

Comment:
is this needed at run time if the user doesn't use the java feature?

Reply:
noup

Reply:
probably should be
```
depends_on :java => ["1.7+", :build]
```

Reply:
or actually it can probably be left out altogether since the jar is prebuilt?

Reply:
which jar? you need java to build the jar

Reply:
so i guess you want to do the same for python?

Reply:
or does the build use `java`?

Reply:
indeed it does

Reply:
>so i guess you want to do the same for python?

It's already the case since macOS comes with Python.

Reply:
>which jar? you need java to build the jar

I'm thinking of `jblas`. I guess it builds more jars?

Reply:
it builds a jar itself, so jvm is required for build

Reply:
OK. What is minimum Java needed? Probably
```
depends_on :java => ["1.7+", :build]
```
?

Reply:
yep, we could go with 1.6 as well but it's EOL so better go with 1.7..... although that one is EOL as well :)